modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,67 +12,66 @@ endif::[]
 
-Supported third party OAuth 2.0 providers:
+You must deploy the Mule OAuth 2.0 provider to a runtime with API gateway capabilities, for example, Mule runtime engine (Mule) versions 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third party OAuth 2.0 providers are supported:
 
 * PingFederate
 * OpenAM
 * OAuth 2.0 providers supporting Open ID Connect
 
-The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] is designed to work with the Mule OAuth 2.0 provider.
+The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] works with the Mule OAuth 2.0 provider.
 
-== Getting the Mule OAuth 2.0 provider
-To run the Mule OAuth 2.0 Provider you have to deploy the provider to a runtime with API Gateway capabilities (E.g. Mule runtime engine v3.x, Mule runtime engine v4.2.0 or later) with the corresponding organization credentials correctly specified.
+== Downloading the Mule OAuth 2.0 Provider
 
-=== Mule v3.x
-You can download the OAuth Provider from Exchange.
+You can download the Mule OAuth 2.0 provider from Anypoint Exchange:
 
-See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-external-oauth2-provider/[External OAuth 2.0 server for Anypoint Platform] asset in Exchange to download it.
-
-=== Mule v4.x
-This feature is available from Mule v4.2.0.
-
-See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset in Exchange to download it.
+* Mule v3.x: See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-external-oauth2-provider/[External OAuth 2.0 server for Anypoint Platform].
+* Mule v4.x: See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
++
+This feature is availble from Mule v4.2.0.
 
 == OAuth 2.0 dance
 
-The authentication process performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This process is called the xref:oauth-dance-about.adoc[OAuth 2.0 dance].
+The authentication performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This authentication process is known as the xref:oauth-dance-about.adoc[OAuth 2.0 dance]. 
+
+*Nahuel--why are we suddenly jumping to grant types and what is a client for the enforcement policy? There seems to be no connection.*
 
 Mule OAuth 2.0 provider supports all xref:oauth-grant-types-about.adoc[grant types].
 
 The entity that requests access to a resource protected by Mule OAuth 2.0 access token enforcement policy is a `Client`.
 
 === Endpoints
 
+*Nahuel--this section is not very clear. What is this section about? What about endpoints? What is it used for? What does it do for the customer?*
```

Comment:
The OAuth provider has URLs that can be called. An endpoint, in Mule parlance, is "An endpoint is the entry point to a service, a process, or a queue or topic destination in service-oriented architecture". So, in order to use this provider, you have to know what it offers, and where. Without this information, the customer has to dig into the code, or not use the provider at all.

Reply:
Added this sentence:
The Mule OAuth 2.0 provider has the following endpoints that you can call in your API implementation:
You "must always" provide an intro text after a heading and before any sub-heading, graphic, or table. I needed the info for that.

Reply:
Ah! Ok. Then I would say something like instead "The Mule OAuth 2.0 provider has the following endpoints that you can call when performing the OAuth 2.0 dance, validating, or revoking a token." The provider endpoints are not related to an API or its implementation. The provider only cares for client applications created within the configured organization/sub organization.